README.md

```diff
@@ -344,7 +344,7 @@ function sendToGoogleAnalytics({name, delta, value, id, attribution}) {
       eventParams.debug_target = attribution.largestShiftTarget;
       break;
     case 'INP':
-      eventParams.debug_target = attribution.eventTarget;
+      eventParams.debug_target = attribution.interactionTargetSelector;
```

Comment:
This example is now a bit unfortunate in its inconsistency. WDYT @tunetheweb?

Reply:
I think it's fine. Some people might choose to use a different definition of `debug_target` (e.g. like Wixs does).

Plus it's inconsistent for LCP too!

https://github.com/GoogleChrome/web-vitals/blob/2cb70da58411016c45411b6a45907eeead9d1fbf/README.md?plain=1#L398

Reply:
Ok, if you're not bothered by it, let's go with this.